markdown/what.md

```diff
@@ -1,23 +1,79 @@
+
+Starting from the Maple release of Open edX, the community-supported method of deploying the platform using Ansible playbooks was deprecated. Instead, the upstream decided to transition towards a containerized deployment approach.
+
+To adapt to this change, we made the switch from OpenStack Heat to Kubernetes on OpenStack Magnum. This shift allowed us to embrace containerization for deploying the Open edX platform, leveraging the benefits it provides in terms of scalability, flexibility, and ease of management.
+
+Alongside the deployment changes, we also had a transition in our CI-driven deployment approach. We moved from Gitlab CI to Zuul.
+ 
+This is a summary of the architecture before and after, which also discusses some of the issues we ran into along the way.
 
 
 ## Open edX
 
 <!-- Note -->
-These are the notes for the next slide.
+
+As mentioned earlier, our platform is built on Open edX. Now let me provide you with a concise overview of what Open edX is and its fundamental components.
+
+Harvard University and MIT collaborated to establish edX, a non-profit organization with a mission to promote an open-source vision for the future of education. In 2013, edX made its code open source, enabling universities, corporations, NGOs, government organizations, and entrepreneurs to utilize this powerful tool.
+
+The edX platform comprises of three main components. Firstly, the Learning Management System (LMS) serves as the application through which learners access and engage with the course materials. The LMS provides a user-friendly interface for learners to view and interact with the educational content.
+
+Secondly, the platform includes Studio, an advanced course creation tool designed for instructors. Studio empowers instructors to develop and design their courses with ease. This sophisticated tool simplifies the process of creating engaging and interactive learning experiences.
+
+Lastly, Django Admin panel, the control house for adminstration and management tasks. It facilitates platform-level changes such as setting permissions and managing data, providing administrators with the tools needed to effectively manage the platform.
+
+All the components within the edX platform are meticulously designed and offer intuitive interfaces, ensuring that users can navigate and utilize the tools effortlessly.
 
 
 ## How we used to run Open edX
 
+<!-- Note -->
+
+When we started building our learning platform on Open edX in 2015, we adopted a deployment strategy that aligned with the community's recommended approach.
+We opted to deploy the platform on cloud server instances using Ansible playbooks.
+
+To achieve a cloud-centric and image-driven deployment, we incorporated OpenStack Heat into our workflow and managed our clusters as Heat stacks. Additionally, we effectively managed the entire system using GitLab CI, leveraging its capabilities to orchestrate the deployment process.
```

Comment:
You can rewrite the second sentence as just "We used GitLab CI to invoke Heat to deploy the platform." Use simple words, cut the fluff. This isn't LinkedIn. :)